templates/repo/settings/lfs_file.tmpl

```diff
@@ -49,6 +57,8 @@
 								</tr>
 							</tbody>
 						</table>
+					{{else if not .FileSize}}
+						<div class="gt-df gt-jc gt-py-4">{{.locale.Tr "empty_file"}}</div>
```

Comment:
I think `text-center` is more readable than `df jc`

Reply:
Agree, should be possible.